src/table/secrets_settings.ini

```diff
@@ -0,0 +1,84 @@
+; This file is part of OpenTTD.
+; OpenTTD is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 2.
+; OpenTTD is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+; See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with OpenTTD. If not, see <http://www.gnu.org/licenses/>.
+;
+
+[pre-amble]
+static bool UpdateServerPassword(int32 p1);
+static bool UpdateRconPassword(int32 p1);
+
+static const SettingTable _secrets_settings = {
+[post-amble]
+};
+[templates]
+SDTC_BOOL  =  SDTC_BOOL(       $var,        $flags, $guiflags, $def,                              $str, $strhelp, $strval, $proc, $from, $to,        $cat, $extra, $startup),
```

Comment:
I wouldn't add templates that are not used yet. That only adds unused code, and if they would ever be needed they can easily be taken from some other location.